app/assets/stylesheets/components/_option-select.scss

```diff
@@ -107,7 +107,7 @@
 .app-c-option-select__filter-input {
   @include govuk-font(19);
   padding-left: 33px;
-  background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 36 36' width='40' height='40'%3E%3Cpath d='M25.7 24.8L21.9 21c.7-1 1.1-2.2 1.1-3.5 0-3.6-2.9-6.5-6.5-6.5S10 13.9 10 17.5s2.9 6.5 6.5 6.5c1.6 0 3-.6 4.1-1.5l3.7 3.7 1.4-1.4zM12 17.5c0-2.5 2-4.5 4.5-4.5s4.5 2 4.5 4.5-2 4.5-4.5 4.5-4.5-2-4.5-4.5z' fill='currentColor'%3E%3C/path%3E%3C/svg%3E") govuk-colour("white") no-repeat -5px -3px;
+  background: image-url("option-select/input-icon.svg") govuk-colour("white") no-repeat -5px -3px;
```

Comment:
Makes sense that inline-images could be a security issue. I guess we can always address the duplication at a later date.